hayeon/programmers-2/다음-큰-숫자.js

```diff
@@ -0,0 +1,18 @@
+function solution(n) {
+  let binaryNum = n.toString(2).split("1").length;
+
+  while (true) {
+    n++;
+    if (n.toString(2).split("1").length === binaryNum) return n;
+  }
+}
```

Comment:
```suggestion
function solution(n) {
  const targetCount = n.toString(2).split("1").length;
  let decimalNum = n;
    
  while (true) {
    const curCount = (++decimalNum).toString(2).split("1").length;
      
    if (curCount === targetCount) {
        return decimalNum;
    };
  }
}
```
- 풀이잘봤습니다
- 기존 binaryNum은 2진수라는 의미인데 해당 코드에서는 의도와 다르다고 생각해서 임시로 targetCount로 바꿨으며, let이 아닌 const로 선언해주셔야 합니다
- 또한 if문에서 조건식이 불필요하게 길어지기 때문에 변수로 빼주는게 가독성에 좋을 것 같습니다. curCount라는 변수명으로 임시로 넣어봤습니다.
- 마지막으로 인자로 받은 n을 그대로 사용하기보다는 따로 지역변수로 선언해서 사용하는 것을 추천드립니다 ~ 
- (let/const...🥲), if문 조건식 길어지면 변수로 빼기 등 좀만 더 기본 컨벤션, 코드 가독성에 대해서 고민해보시면 좋을 것 같습니다